wildwood/_tree.py

```diff
@@ -155,6 +158,7 @@ def __init__(self, n_features, n_classes, random_state):
         self.bin_partitions = np.empty(0, dtype=np.uint8)
         self.bin_partitions_capacity = 0
         self.bin_partitions_end = 0
+        self.cat_split_strategy = 0
```

Comment:
I think that this option should go into TreeContext only, not in the TreeClassifier and TreeRegressor (it only contains data)

Reply:
It is actually not needed in TreeRegressor and in the current state of the code, I don't think it can be removed from TreeClassifier since the latter only creates its TreeContext object when its fit method is called